sound/soc/sof/ipc4-topology.c

```diff
@@ -844,6 +859,17 @@ static void sof_ipc4_unprepare_copier_module(struct snd_sof_widget *swidget)
 		struct snd_sof_dai *dai = swidget->private;
 
 		ipc4_copier = dai->private;
+		if (ipc4_copier->dai_type == SOF_DAI_INTEL_ALH) {
+			struct sof_ipc4_alh_configuration_blob *blob;
+			u32 group_id;
```

Comment:
Can we use the same data type here and in prepare for group_id?

Reply:
> Can we use the same data type here and in prepare for group_id?

No, the prototype of `ida_free` is `void ida_free(struct ida *ida, unsigned int id)`. Besides, group_id here is a part of gtw_cfg.node_id which is `uint32_t`